services/swagger.js

```diff
@@ -131,6 +134,34 @@ exports.addFormat = function (format, validationFunction) {
     tv4.addFormat(format, validationFunction);
 };
 
+function overrideInheritedProperties(definitions) {
```

Comment:
I think we want a less scary name for this function (the current one makes it seem like this is something we're just doing with no connection to what the user wanted)

some possible alternates:
- `applySubmodelProperties`
- `resolveSubmodelProperties`
- `applySubmodelPropertyOverrides`
- `applySubmodelPropertyCustomization`